NewSpring/corinth
# fix: Fixed the liked content feed to navigate correctly

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR fixes an issue where viewing a piece of content from the liked content feed would crash the app.

### How do I test this PR?
1. Run the app on the `master` branch.
* Before starting, make sure that you have a piece of content liked.
* Go to the Connect tab
* Tap "View All" next to the "Recently Liked" section to take you to a feed of your liked content.
* Tap on one of the items in that feed. 
* The app will error.

2. Run the app on this `fix-liked-content` branch.
* Follow the same steps above. When you tap on the content item it should correctly take you to the item without error.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
